src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-application-build-number.mdx

```diff
@@ -0,0 +1,175 @@
+---
+title: Set application build ID
+type: apiDoc
+shortDescription: 'Set a custom build identifier string that is associated with an interaction.'
+tags:
+  - Mobile monitoring
+  - Mobile SDK API
+  - Custom instrumentation
+metaDescription: Mobile app monitoring API used to set a custom build identifier string that is associated with an interaction.
+redirects:
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/withapplicationbuild-android-agent-api
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/withapplicationbuild-android-sdk-api
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/with-application-build
+  - /docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/set-application-build-ios-sdk-api
+  - /docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/set-application-build
+---
+
+<Tabs>
+	<TabsBar>
+        <TabsBarItem id="android">
+            Android
+        </TabsBarItem>
+        <TabsBarItem id="ios">
+            iOS
+        </TabsBarItem>
+    </TabsBar>
+
+    <TabsPages>
+        <TabsPageItem id="android">
+
+<Callout variant="important">
+  As of version 5.3.0 of the New Relic Android agent, `withApplicationBuild()` has replaced `NewRelic.withBuildIdentifier()`.
+  </Callout>
+  
+## Syntax [#syntax]
+
+### Java [#java]
+
+```java
+NewRelic.withApplicationBuild(string $buildId)
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
+NewRelic.withApplicationBuild(buildId: String?)
+```
+
+## Description [#description]
+
+This call allows you to set a custom build identifier string that appears next to the application version in the [**Crash report** page](/docs/mobile-monitoring/mobile-monitoring-ui/crashes/mobile-apps-crash-report-dashboard#details). Instead of using the `versionCode` string defined in `AndroidManifest.xml`, call the `withApplicationBuild()` method and pass along the custom build ID as a string.
+
+## Parameters [#parameters]
+
+<table>
+  <thead>
+    <tr>
+      <th width={200}>
+        Parameter
+      </th>
+      <th width={200}>
+        Type
+      </th>
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        `$buildId`
+      </td>
+
+            <td>
+        `string`
+      </td>
+
+      <td>
+        Required. The string indicating build ID.
+      </td>
+    </tr>
+
+  </tbody>
+</table>
+
+
+## Example [#example]
+
+### Java [#java]
+
+```java
+NewRelic.withApplicationBuild('156')
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
+
+```
```

Comment:
@ndesai-newrelic Can you please provide a kotlin example equivalent to the Java example?

Reply:
Please Change Java Example also.

1) Java Example 
        NewRelic.withApplicationToken("<NewRelic App Token>").withApplicationBuild("1").start(this.getApplication());

2)         NewRelic.withApplicationToken("<NewRelic App Token>").withApplicationBuild("1").start(applicationContext)



2) Kotlin Example:

Reply:
```suggestion
```kotlin

NewRelic.withApplicationToken("").withApplicationBuild("1").start(applicationContext)

```
```